src/Patient/Cards/CareTeamViewCard.php

```diff
@@ -0,0 +1,293 @@
+        $teamName = '';
+        while ($member = sqlFetchArray($careTeamResult)) {
+            $existingCareTeam[] = [
+                'user_id' => $member['user_id'],
+                'role' => $member['role'],
+                'facility_id' => $member['facility_id'],
+                'provider_since' => $member['provider_since'],
+                'status' => $member['status'],
+                'note' => $member['note'],
+                'user_name' => trim(($member['lname'] ?? '') . ", " . ($member['fname'] ?? ''))
+            ];
+
+            if (empty($teamName) && !empty($member['team_name'])) {
+                $teamName = $member['team_name'];
+            }
+        }
+        // Enhanced template data with pre-generated option strings for JavaScript
+        $templateData['user_options'] = '';
+        foreach ($templateData['users'] as $users) {
+            $templateData['user_options'] .= "<option value='" . attr($users['id']) . "'>" . text($users['name']) . "</option>";
+        }
+
+        $templateData['facility_options'] = '';
+        foreach ($templateData['facilities'] as $facilities) {
+            $templateData['facility_options'] .= "<option value='" . attr($facilities['id']) . "'>" . text($facilities['name']) . "</option>";
+        }
+
+        $templateData['role_options'] = '';
+        foreach ($templateData['roles'] as $roles) {
+            $templateData['role_options'] .= "<option value='" . attr($roles['id']) . "'>" . text($roles['title']) . "</option>";
+        }
+
+        $templateData['status_options'] = '';
+        foreach ($templateData['statuses'] as $statuses) {
+            $templateData['status_options'] .= "<option value='" . attr($statuses['id']) . "'>" . text($statuses['title']) . "</option>";
+        }
+
+        return [
+            'pid' => $pid,
+            'team_name' => $teamName,
+            'user_options' => $templateData['user_options'],
+            'facility_options' => $templateData['facility_options'],
+            'role_options' => $templateData['role_options'],
+            'status_options' => $templateData['status_options'],
+            'existing_care_team' => $existingCareTeam,
+            'csrf_token' => CsrfUtils::collectCsrfToken(),
+            'translations' => self::getTranslations()
+        ];
+    }
+
+    private static function getTranslations()
+    {
+        return [
+            'manage_care_team' => xlt("Manage Care Team"),
+            'care_team_name' => xlt("Care Team Name"),
+            'member' => xlt("Member"),
+            'role' => xlt("Role"),
+            'facility' => xlt("Facility"),
+            'since' => xlt("Since"),
+            'status' => xlt("Status"),
+            'note' => xlt("Note"),
+            'remove' => xlt("Remove"),
+            'add_team_member' => xlt("Add Team Member"),
+            'save_care_team' => xlt("Save Care Team"),
+            'save_care_team_confirm' => xlt('Save care team?')
+        ];
```

Comment:
I would only translate and not escape here to give more flexibility on how you will be escaping (ie. text, attr, js_escape)